app/templates/components/public/call-for-speakers.hbs

```diff
@@ -1,32 +1,15 @@
 <h1 class="ui header">
-  {{t 'Call for Speakers'}}
-  {{#if callForSpeakers.isClosed}}
+  {{speakersCall.announcement}}
+  {{#if is-after eventDetails.endsAt speakersCall.startsAt precision='year'}}
     <a class="ui basic red label">{{t 'Closed'}}</a>
   {{else}}
     <a class="ui basic green label">{{t 'Open'}} </a>
     <div class="sub header">
-      {{t 'Call for Speakers Open until'}} Fri, Oct 13 2017 5:00 pm
+      {{t 'Call for Speakers Open until'}} {{moment-format speakersCall.endsAt 'ddd, MMM DD HH:mm A'}}
     </div>
   {{/if}}
 </h1>
 <div class="ui divider"></div>
-  <p>
```

Comment:
Announcement should be here. And it will have html content. So use `sanitize` helper